README.md

```diff
@@ -13,13 +13,86 @@
 - [Go](https://golang.org/doc/install) 1.22.0 (to build the provider plugin)
 
 
+## Migration to v1
+
+> [!WARNING]
+>
+> Before you begin using the ```v1``` binary on your Terraform code, make sure to back up your state file!
+> If you are using a local state file, you can just copy of your terraform.tfstate file in your project directory.
+>
+> If you are using a remote backend such as an S3 bucket, make sure that you follow the backup procedures and that you exercise the restore procedure at least once.
+>
+> Additionally, make sure you backup or version your code as migration will require some code changes (on Flexible_gpu resource).
+
+### Step 1: Upgrade provider version
+
+```sh
+terraform {
+  required_providers {
+    outscale = {
+      source = "outscale/outscale"
+      version = "1.0.0-rc.1"
+    }
+  }
+}
+
+provider "outscale" {
+  # Configuration options
+}
+```
+
+```sh
+terraform init -upgrade
+```
+
+### Step 2: Edit terraform state and configuration files
+
+Somes block types changed in terraform state, the following script will delete those blocks.
```

Comment:
Some* block types changed in the Terraform state. The following script will delete those blocks.